win/CS/HandBrakeWPF/ViewModels/VideoViewModel.cs

```diff
@@ -251,7 +249,11 @@ public int RF
                     case VideoEncoder.MFH265:
                         double cqStep = userSettingService.GetUserSetting<double>(UserSettingConstants.X264Step);
                         this.Task.Quality = Math.Round(51.0 - (value * cqStep), 2);
-                        break; 
+                        break;
+                    case VideoEncoder.QuickSync:
+                    case VideoEncoder.QuickSyncH265:
+                        this.Task.Quality = Math.Round(51.0 - (value - 0), 0);

```

Comment:
I'll need to check what's going on, but we should be able to allow the step granularity for QSV as well to allow more fine grained control.  I.e. 21.5

Reply:
mfxU16 ICQQuality is an unsigned integer value.
https://github.com/Intel-Media-SDK/MediaSDK/blob/master/doc/mediasdk-man.md#mfxinfomfx

Reply:
bummer. That's a shame.  Fair enough then!